src/bio/bio_wal.c

```diff
@@ -1985,13 +1997,15 @@ load_meta_header(struct bio_meta_context *mc)
 		return -DER_UNINIT;
 	}
 
-	if (hdr->mh_version != BIO_META_VERSION) {
+	if (hdr->mh_version != BIO_META_VERSION && hdr->mh_version != 1) {
```

Comment:
I would recommend to create a `BIO_META_VERSION_1` define instead of using raw numbers.

Reply:
Sure, I can make such change in following PRs, otherwise, this PR will have to take another few months to pass CI. :)